spec/swagger.json

```diff
@@ -3,7 +3,7 @@
   "info": {
```

Comment:
when I run this in dapperbox I see the generateFinanceReports under the payment service section

![Screenshot 2020-10-15 at 11 47 32](https://user-images.githubusercontent.com/2736331/96113644-5f560900-0edc-11eb-97a2-79e0e45b8342.png)

Reply:
not a biggy